libenkf/src/enkf_main.c

```diff
@@ -1538,7 +1538,7 @@ static void enkf_main_start_queue(enkf_main_type * enkf_main,
   enkf_main_submit_jobs( enkf_main , run_context, job_queue);
 
   job_queue_submit_complete( job_queue );
-  res_log_add_message( 1 , NULL , "All jobs submitted to internal queue - waiting for completion" ,  false);
+  res_log_add_message_str(LOG_ERROR , "All jobs submitted to internal queue - waiting for completion");
```

Comment:
No, it certainly does not. A result of the simple/braindead numeric->semantic conversion I did.